src/shared/experiments/experiments.json

```diff
@@ -0,0 +1,8 @@
+{
+    "experiments": [
+        {
+            "id": "jsonResourceModification",
+            "description": "Basic JSON-based create, update, and delete support for cloud resources in the Explorer"
```

Comment:
Why isn't this driven by the vscode settings?

If we can't use vscode settings for some reason, why does this need to be JSON? It could just be a typescript data structure.